src/modelmanager.hpp

```diff
@@ -77,6 +77,9 @@ class ModelManager {
      */
     ModelManager(const ModelManager&) = delete;
 
+    static Status lastLoadConfigStatus;
```

Comment:
Since we are using a lot of ModelManagers in unit tests unless there is good reason I would stick with object member instead of static class member.